parameter/ParameterMgr.cpp

```diff
@@ -519,7 +536,12 @@ bool CParameterMgr::loadFrameworkConfiguration(string& strError)
     // Parse Structure XML file
     CXmlElementSerializingContext elementSerializingContext(strError);
 
-    if (!xmlParse(elementSerializingContext, getFrameworkConfiguration(), _strXmlConfigurationFilePath, _strXmlConfigurationFolderPath, EFrameworkConfigurationLibrary)) {
+    const ifstream file(_strXmlConfigurationFilePath.c_str());
+    if (!file) {
+        strError = "Failed to open " + _strXmlConfigurationFilePath + " for reading.";
+        return false;
+    }
+    if (!xmlParse(elementSerializingContext, getFrameworkConfiguration(), file, _strXmlConfigurationFolderPath, EFrameworkConfigurationLibrary)) {
```

Comment:
no need to: it is done by xmlParse through `elementSerializingContext.setError(...)`